rest/data/project.go

```diff
@@ -75,6 +80,39 @@ func CreateProject(ctx context.Context, projectRef *model.ProjectRef, u *user.DB
 			return err
 		}
 	}
+
+	cachedSettings := env.Settings()
```

Comment:
Curious why we are caching the value rather than passing a `env.Settings()` into each function?

Reply:
The name is to distinguish from the admin settings that are loaded on L58 using `evergreen.GetConfig()`, which returns the settings directly from the DB, as opposed to `env.Settings()`, which returns the settings as they existed when the app server first started.